routes/web.php

```diff
@@ -41,12 +41,14 @@
     URL::forceRootUrl(config('unit3d.root_url_override'));
```

Comment:
Why are you creating a new routes file. These should be in mediahub group in the web routes file along with rest.